README.rst

```diff
@@ -50,31 +48,29 @@ Also included are SQL drivers:
 |Hibernate5        |Java 8 or 11         |
 +------------------+---------------------+
 
-Of course, these clients require a NuoDB database to be running.  If you don't
-have NuoDB_ available yet, you can use the `NuoDB Community Edition`_.
+The use of the client drivers require a running NuoDB database.  If you don't
+have a NuoDB_ database running in your environment yet, you can use the NuoDB `Community Edition`_ page to guide you through the steps to deploy a NuoDB database.
```

Comment:
Please note I am trying to keep all the lines in the readme hard-wrapped at about 72-75 characters long to fit easily into a standard terminal window.